doc/devel/coding_guide.rst

```diff
@@ -165,6 +165,48 @@ C/C++ extensions
 
 
 
+PR Review guidelines
+====================
+
+* We run on a 'use your judgement' system (for better or worse) and do
+  not have a strict nose-count rule.  If you have a commit bit, then
```

Comment:
You can edit the PR title before merging.
I personally don't like labels. They are hard to filter, clutter the UI, and half of them are IMO useless. They are also slower to edit (to move from MRG+1 to MRG+2 would require to remove a label and add a new one, or keep the two, but that's a lot of useless information displayed to the user).

Stéfan van der Walt (one of the main maintainers of scikit-image) recently strongly advocated the new projects features of github for classifying PRs into different categories. I haven't used it yet, but I can give feedback in a month or so once I'll use it on common projects with him.

The scikit-learn team is sticking to editing the PR title as we found it is, right now, the most efficient way to scan quickly through PRs to have an overview of the WIP (work in progress), and the different review states of PRs. But then our commit logs look like this: https://github.com/scikit-learn/scikit-learn/commits/master (which I am personally fine with)